### ISSUE TITLE
recurring events

### ISSUE THREAD

**Issue by HanEmile** [pretalx/pretalx#1423]
### Problem and impact

So we've got this weekly event at our hackspace with three 20 min. talks. People can currently submit talks via our wiki and the whole process is rather hacky. We'd like to use pretalx, but there are often talks that are submitted for one week that are then moved to the next week. This is currently not really possible within pretalx (or I haven't found it yet!).

### Proposed solution / feature

Ideally, people should be able to move their talks from one event to another.

### Context or examples

Say you submit a talk to event "A", but can't make it and want to hold the talk on event "B". You should be able to move the talk or event export and reimport it easily.

**Comment by nomaster** [comment]
This is probably useful for a lot of interest groups meeting regularly - called "Stammtisch" in Germany.

Another approach would be the possibility for an event that goes on forever (no end date), with the rooms opening on a certain day of the month.

**Comment by aNullValue** [comment]
I agree that it would be an awesome feature to be able to move (or copy) a talk from one event to another. I'm concerned about doing so by controlling room open though. An evaluation would need to be done of the effects on the presentation of the schedule in Pretalx and any other apps that might be displaying it. For example, HackerTracker (an app that displays the conference schedule by consuming the Frab json export) wouldn't be able to present that well, because it means the conference would permanently be "happening now". I suspect (but have not verified) that other apps consuming the schedule would behave similarly.

**Comment by rixx** [comment]
I don't think that recurring events are realistically in pretalx's future any time soon. However, if we make it easy to move proposals to different events, you could still have one ongoing CfP, and then move talks to specific events. As you can already copy one event's configuration to the next, I feel like that would be pretty close to what you want, while still being reasonable to build and maintain.
